src/Main.js

```diff
@@ -1,21 +1,24 @@
 // Modules using `default` export must be imported...
 import * as THREE from 'three';
 import proj4 from 'proj4';
-import ApiGlobe, { INITIALIZED_EVENT } from './Core/Scheduler/Interfaces/ApiInterface/ApiGlobe';
 import { CONTROL_EVENTS } from './Renderer/ThreeExtended/GlobeControls';
 import View from './Core/View';
-import GlobeView from './Core/Prefab/GlobeView';
+import GlobeView, { GLOBE_VIEW_EVENTS } from './Core/Prefab/GlobeView';
+import { ColorLayersOrdering } from './Core/Prefab/ColorLayersOrdering';
```

Comment:
You can replace this line and the matching export below by:
`export { ColorLayersOrdering } from './Core/Prefab/ColorLayersOrdering'; `